src/Microsoft.AspNetCore.SignalR.Client/LongPollingTransport.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.IO.Pipelines;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.SignalR.Client
+{
+    public class LongPollingTransport : ITransport
+    {
+        private static readonly string DefaultUserAgent = "Microsoft.AspNetCore.SignalR.Client/0.0.0";
```

Comment:
preparing requests should be common. Even now this is not being sent when requesting a connection id.